deploy-tool/cert/cert.go

```diff
@@ -7,10 +7,11 @@ import (
 	"crypto/rand"
 	"crypto/x509"
 	"fmt"
+	"strings"
+
 	"github.com/abetterinternet/prio-server/deploy-tool/config"
 	"github.com/abetterinternet/prio-server/deploy-tool/dns"
 	"github.com/caddyserver/certmagic"
-	"strings"
```

Comment:
I like the reorganizing of the `import` statements. Beyond splitting stdlib stuff from `github.com` deps, what I've also frequently seen is putting deps from our own modules in a separate block, but I'm not sure if that's formally part of Go style.